build_targets

```diff
@@ -7,7 +7,7 @@ beryllium userdebug yes [recovery.img]
 #courbet user yes yes
 #dumpling user yes no
 guacamole userdebug yes [boot.img]
-#hotdog userdebug yes yes
+hotdog userdebug yes [recovery.img, dtbo.img, vbmeta.img]
```

Comment:
shouldn't be boot instead of recovery?

Reply:
No. We have a separate partition - recovery

Reply:
guacamole uses boot, reason I asked

Reply:
Lineage recommended dtbo and vbmeta - https://wiki.lineageos.org/devices/hotdog/install/#flashing-additional-partitions

Reply:
> guacamole uses boot, reason I asked

Yes, I remember. We once discussed this in the Telegram